homework/add-even/addEven.hpp

```diff
@@ -1,8 +1,13 @@
 #pragma once
 #include <vector>
+#include <algorithm>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int sum {0};
+    std::for_each(numbers.begin(), numbers.end(), [&sum](int element) {
+                 if(element % 2 == 0) {
+                    sum += element;
+                 }
+            });
```

Comment:
I know for_each is universal loop, but more readable will be simple for loop:

```C++
for (const auto& el : vec) {
}
```

Because you don't need to use iterator and lambda :)